README.md

```diff
@@ -24,9 +24,18 @@ See [CONTRIBUTING.md](/CONTRIBUTING.md) for ways to get started.
 - [dnd kit](https://dndkit.com/)
 - [react-markdown](https://github.com/remarkjs/react-markdown)
 
+## Installing/Updating NextJS:
+First step is being sure your running the latest [NEXTJS](https://github.com/vercel/next.js):
+
+`npm install next` or `npm update next`
+
+Most likely you should just be able to update here instead of installing as it should already be installed.
```

Comment:
What does "here" refer to in this sentence? That isn't very clear to me.

Also, I'd suggest adding a comma after "installing", to make the sentence structure more evident.